### PR TITLE
Hannahnjoroge/updatereadme

### PR DESCRIPTION
**Pull request by HWaruguru**
Updates the README file to add basic information. 
[Task](https://phabricator.wikimedia.org/T324644): T324644

### CODE REVIEW

**Comment by HWaruguru** [comment]
great I am still waiting for one more approval so that we can proceed. Do we also add the license at this point?

File: README.md
Comment (blancadesal) [inline README.md:24]: Bootstrap: As we haven't discussed component libraries yet, I'd leave this out for now.

File: README.md
Comment (blancadesal) [inline README.md:5]: Could you add link to Outreachy?

File: README.md
Comment (blancadesal) [inline README.md:8]: I'd be more explicit about us using Phabricator for issue tracking (right now it's hidden in the link), and also advising against using GitHub's own issue tracker for bugs, etc. (There may actually be a way to disable GH issues from the settings – I'll investigate).

**Review by blancadesal — CHANGES_REQUESTED** [review]
LGTM overall; some small comments inline.

**Comment by blancadesal** [comment]
> great I am still waiting for one more approval so that we can proceed. Do we also add the license at this point?

The license would be a separate PR.

**Review by blancadesal — APPROVED** [review]

**Comment by HWaruguru** [comment]
Can I merge now?

**Pull request merged**
